src/components/BrowseSkill/BrowseSkill.js

```diff
@@ -134,6 +135,13 @@ export default class BrowseSkill extends React.Component {
     });
   };
 
+  handleShowSkills = (event, index, value) => {
```

Comment:
event and index params are not in use anywhere

Reply:
@anshumanv This is the standard parameter way of onChange() function of SelectField component. `event` signifies the target click event, which isn't explicitly used, but is required. If I remove `index`, then `value` would be treated as the second parameter and hence, this wouldn't work. You can also check the other filtrations, those are implemented in the same way. Thanks :-)

Reply:
Okay cool. :+1: